packages/fury-adapter-oas3-parser/lib/parser/oas/parseSecurityRequirementObject.js

```diff
@@ -40,16 +40,17 @@ function parseSecurityRequirementObject(context, object) {
   const parseSecurityRequirement = pipeParseResult(namespace,
     parseObject(context, name, parseMember),
     (securityRequirement) => {
+      // TODO: expand oauth requirements into multiples depending on flows
```

Comment:
This is the only thing that's not implemented in the whole of security area. If the swagger specifies an oauth2 based security scheme, we are expanding it into multiple security schemes so that we don't break the APIE regarding flows. So, when the user uses that security scheme in a security requirement, we need to replace that security requirement with the multiple security schemes we created above.

Reply:
Part of next PR